src/server/test/unit/liquidTagGrammer.test.ts

```diff
@@ -0,0 +1,330 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/*!
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ */
+
+import { expect } from 'chai';
+import * as nearley from 'nearley';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const grammar = require('../../Parser/liquidTagGrammar');
+
+describe('LiquidGrammar', () => {
+
+
+  //#region "Scenarios for Entity Form"
```

Comment:
there are no #regions in VSCode. If you want to create structure of cohesive it() clauses, that's what the describe(string) is for

Reply:
_one_